scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -0,0 +1,46 @@
+package com.spotify.scio
+
+import org.apache.beam.sdk.io.jdbc.JdbcIO.{PreparedStatementSetter, RowMapper, StatementPreparator}
+
+package object jdbc {
+
+  val TEST_READ_TABLE_NAME = "table_read"
+  val TEST_WRITE_TABLE_NAME = "table_write"
+
+  /**
+    * Options require to create a connection with remote database.
+    * @param username database username
+    * @param password database password
+    * @param connectionUrl connection url i.e "jdbc:mysql://[host]:[port]/db?"
+    * @param driverClassName database driver class name
+    */
+  case class DbConnectionOptions(username: String,
+                                 password: String,
+                                 connectionUrl: String,
+                                 driverClassName: String)
```

Comment:
How about accepting a `Class` object instead of `String` for `driverClassName` just to mitigate a simple mistake of typos in `driverClassName`?